src/main/java/frc/robot/subsystems/gpm/Arm.java

```diff
@@ -235,16 +226,16 @@ public Arm() {
 
     @Override
     public void periodic() {
-
+        SmartDashboard.putNumber("get Position", getPosition());
 
         // Disable the arm if it is out of range
 		if (getAngleRad() < ArmConstants.MIN_ANGLE_RADS - ArmConstants.ANGLE_TOLERANCE || getAngleRad() > ArmConstants.MAX_ANGLE_RADS + ArmConstants.ANGLE_TOLERANCE) {
-			System.err.println("WARNING: THE ARM IS IN A SUPPOSEDLY UNREACHABLE POSITION AND HAS BEEN DISABLED. Found: " + getAngleRad() + ", Expected: " + ArmConstants.stowedSetpoint);
+			System.err.println("WWARNING: THE ARM IS IN A SUPPOSEDLY UNREACHABLE POSITION AND HAS BEEN DISABLED. Found: " + getAngleRad() + ", Expected: " + ArmConstants.stowedSetpoint);
 			for (int i = 0; i < motors.length; i++) {
-				motors[i].setNeutralMode(NeutralModeValue.Coast);
-				motors[i].set(0);
+				// irrelevant for next line: motors[i].setNeutralMode(NeutralModeValue.Coast);
+				//motors[i].set(0);
+                System.err.println("hwat");
 			}
```

Comment:
Don't send the command to the master and the slaves. Sending it to the master should be enough.